scenarios/proxy.grpc.benchmarks.yml

```diff
@@ -0,0 +1,130 @@
+﻿variables:
+    path: /
+    serverScheme: https # http, https
+    downstreamScheme: https # http, https
+    serverPort: 8080
+    downstreamPort: 5000
+    connections: 1
+    streams: 100
+    grpcClientType: GrpcNetClient
+    scenario: serverstreaming
+    protocol: h2
+    warmup: 5
+    duration: 5
+    logLevel: none
+    requestSize: 0
+    responseSize: 0
+    enableCertAuth: false
+    latency: false
+
+jobs:
+  grpcDotNetClient:
```

Comment:
You didn't want to import the existing yaml files that contain these definitions?

Reply:
I tried, but it didn't work properly. Please, check the above comment.